autopush/web/message.py

```diff
@@ -9,7 +10,7 @@
 
 class MessageSchema(Schema):
     uaid = fields.UUID()
-    channel_id = fields.UUID(allow_none=True)
+    channel_id = fields.Str(allow_none=True)
```

Comment:
Changing this to str removes the validation that occurs when requiring it to be a valid UUID.